src/micro_http/src/response.rs

```diff
@@ -77,7 +77,7 @@ impl StatusLine {
 /// When creating a ResponseHeaders object, the content type is initialized to `text/plain`.
 /// The content type can be updated with a call to `set_content_type`.
 #[derive(Debug, PartialEq)]
-pub struct ResponseHeaders {
+struct ResponseHeaders {
```

Comment:
I wouldn't remove this particular `pub` since this crate may be individually consumed and there is a use case where a user needs to view and access the `ResponseHeaders` struct. Consider keeping the struct public and exporting it through the `lib.rs` file in `micro-http`.

Reply:
ack. updated as suggested.